contrib/epee/include/storages/portable_storage.h

```diff
@@ -24,24 +24,15 @@
 // SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 // 
 
-
-
 #pragma once 
 
-#include <type_traits>
-
-#include "misc_language.h"
-#include "portable_storage_base.h"
-#include "portable_storage_to_bin.h"
-#include "portable_storage_from_bin.h"
-#include "portable_storage_to_json.h"
-#include "portable_storage_from_json.h"
 #include "portable_storage_val_converters.h"
-#include "span.h"
-#include "int-util.h"
+#include "portable_storage_base.h"
+#include "misc_log_ex.h"
```

Comment:
I would better sort it alphabetically

Reply:
Header order has impact on the ease of usage of the classes / libraries, that you define. It's nicely described in this short blog post:
https://blog.knatten.org/2010/07/01/the-order-of-include-directives-matter/

Reply:
Ordered alphabetically.